apollo-integration/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.ben_schwab.topshelf.apollointegration">
+
+  <application android:allowBackup="true" android:label="@string/app_name" android:supportsRtl="true">
```

Comment:
my understanding of `android:allowBackup` flag is that if any manifest that is being merged has that flag set then the client app will have it no matter what their main manifest says.  Our internal policy is to have libraries define android:allowBackup="false" so that client apps can still overwrite it.